flink/v1.19/flink/src/test/java/org/apache/iceberg/flink/source/TestIcebergSourceContinuous.java

```diff
@@ -392,7 +392,8 @@ public void testSpecificSnapshotTimestamp() throws Exception {
   public void testReadingFromBranch() throws Exception {
     String branch = "b1";
     GenericAppenderHelper dataAppender =
-        new GenericAppenderHelper(tableResource.table(), FileFormat.PARQUET, TEMPORARY_FOLDER);
+        new GenericAppenderHelper(
+            tableResource.table(), FileFormat.PARQUET, TEMPORARY_FOLDER.getRoot().toPath());
```

Comment:
is this change still necessary?

Reply:
Nope, reverted. This was a leftover. I double checked, now this is removed from everywhere.